src/Common/test-runtime/project.json

```diff
@@ -98,7 +98,7 @@
         "OpenCover": "4.6.519",
         "ReportGenerator": "2.4.3",
         "Microsoft.NETCore.TestHost": "2.0.0-preview2-25304-01",
-        "Microsoft.NETCore.App": "2.0.0-preview2-25310-02",
+        "Microsoft.NETCore.App": "2.1.0-preview1-25323-04",
```

Comment:
This should be a latest build of `2.0.0-preview2-xxxx` instead for the 2.0 branch.